db/seeds/source_types.yml

```diff
@@ -264,6 +264,20 @@ openshift:
           :is: true
   :vendor: Red Hat
   :icon_url: "/apps/frontend-assets/platform-logos/openshift-container-platform.svg"
+rh-marketplace:
+  :product_name: Red Hat Marketplace
+  :vendor: Red Hat
+  :schema:
+    :authentication:
+      - :type: token
+        :name: API Token
+        :fields:
+          - :component: text-field
+            :name: application.extra.api_token
```

Comment:
Shouldn’t we store the token as a password (so it stays encrypted) in the authentication object? Cc @lindgrenj6

Reply:
You are right, yes. Thanks!